### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by Lewiky**
Implements the README describing how to use Brew to install Temple

Based on https://github.com/facebook/homebrew-fb

### CODE REVIEW

**Comment by c-harding** [comment]
That was quick

**Comment by Lewiky** [comment]
theres an extra commit here that shouldn't be

File: README.md [hunk 1]
```diff
@@ -1,2 +1,69 @@
-# homebrew-temple
-Homebrew Formulae for Temple
+TempleEight Homebrew Tap
+=====================
+
+This is a [Homebrew][brew] tap for formulae for software developed by TempleEight.
```
Comment (xsanda) [inline README.md]: How does this link even work?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,69 @@
...
+
+
+Setup
+-----
+
+Using these formulae requires Homebrew, which in turn requires Xcode. If you
```
Comment (c-harding) [inline README.md]: Not full Xcode

File: README.md [hunk 3]
```diff
@@ -1,2 +1,69 @@
...
+have not yet installed Homebrew, a quick summary is at the end of this
+document.
+
+Once homebrew is installed, simply run:
+
+    brew tap templeeight/temple
+
+Some of these formulae may require OS X 10.10 (Yosemite) or higher.
```
Comment (c-harding) [inline README.md]: Some of which formulae? Can we test?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,69 @@
...
+To install software, just use `brew install` with the name of the formula. You
+may wish to run `brew update` before hand to get the latest version of the
+formulae. For example, to install the latest version of Temple:
+
+    brew update
+    brew install temple
+
+or to install Temple from the latest HEAD commit:
+
+    brew install --HEAD temple
```
Comment (c-harding) [inline README.md]: HEAD of which repo? How does this work?

File: README.md [hunk 5]
```diff
@@ -1,2 +1,69 @@
...
+
+To upgrade software:
+
+    brew update
+    brew upgrade    # upgrade all software installed with Homebrew
```
Comment (c-harding) [inline README.md]: Can we put the explanations in Markdown rather than shell comments?

File: README.md [hunk 6]
```diff
@@ -1,2 +1,69 @@
...
+    echo '$PATH="/opt/homebrew/bin:/opt/homebrew/sbin:$PATH"' >> ~/.bashrc
+
+
+References
+----------
+`brew help`, `man brew`, or the Homebrew [documentation][].
+
+[brew]: http://brew.sh/
+[style]: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md
+[documentation]: https://github.com/Homebrew/homebrew/tree/master/share/doc/homebrew#readme
```
Comment (c-harding) [inline README.md]: 404

File: README.md [hunk 7]
```diff
@@ -1,2 +1,69 @@
...
+
+    brew upgrade 
+
+To update just temple:
+
+    brew upgrade temple  
+
+
+Appendix: overview of installing Homebrew
+-----------------------------------------
```
Comment (c-harding) [inline README.md]:
```suggestion
## Appendix: overview of installing Homebrew
```

File: README.md [hunk 8]
```diff
@@ -1,2 +1,69 @@
-# homebrew-temple
-Homebrew Formulae for Temple
+TempleEight Homebrew Tap
+=====================
```
Comment (c-harding) [inline README.md]:
```suggestion
# TempleEight Homebrew Tap
```

File: README.md [hunk 9]
```diff
@@ -1,2 +1,69 @@
...
+
+This is a [Homebrew][brew] tap for formulae for software developed by TempleEight.
+
+
+Setup
+-----
```
Comment (c-harding) [inline README.md]:
```suggestion
## Setup
```

File: README.md [hunk 10]
```diff
@@ -1,2 +1,69 @@
...
+have not yet installed Homebrew, a quick summary is at the end of this
+document.
+
+Once homebrew is installed, simply run:
+
+    brew tap templeeight/temple
+
+
+Use
+---
```
Comment (c-harding) [inline README.md]:
```suggestion
## Use
```

File: README.md [hunk 11]
```diff
@@ -1,2 +1,69 @@
...
+    sudo chown `whoami` /opt/homebrew
+    curl -sSLf -o homebrew-installer https://raw.githubusercontent.com/Homebrew/install/master/install
+    perl -pi -e s,/usr/local,/opt/homebrew, homebrew-installer
+    ruby homebrew-installer
+    rm homebrew-installer
+    echo '$PATH="/opt/homebrew/bin:/opt/homebrew/sbin:$PATH"' >> ~/.bashrc
+
+
+References
+----------
```
Comment (c-harding) [inline README.md]:
```suggestion
## References
```

File: README.md [hunk 12]
```diff
@@ -1,2 +1,69 @@
...
+To install software, just use `brew install` with the name of the formula. You
+may wish to run `brew update` before hand to get the latest version of the
+formulae. For example, to install the latest version of Temple:
+
+    brew update
+    brew install temple
+
+To upgrade software:
+
+    brew update
```
Comment (c-harding) [inline README.md]: This does not upgrade software

**Review by c-harding — COMMENTED** [review]
My main issue is the label of `brew update`, otherwise good to go

File: README.md [hunk 13]
```diff
@@ -1,2 +1,65 @@
...
+    sudo mkdir /opt/homebrew
+    sudo chown `whoami` /opt/homebrew
+    curl -sSLf -o homebrew-installer https://raw.githubusercontent.com/Homebrew/install/master/install
+    perl -pi -e s,/usr/local,/opt/homebrew, homebrew-installer
+    ruby homebrew-installer
+    rm homebrew-installer
+    echo '$PATH="/opt/homebrew/bin:/opt/homebrew/sbin:$PATH"' >> ~/.bashrc
+
+
+##References
```
Comment (c-harding) [inline README.md]: Look at GitHub’s view of this file, you need a space

File: README.md [hunk 14]
```diff
@@ -1,2 +1,65 @@
...
+## Use
+
+To install software, just use `brew install` with the name of the formula. You
+may wish to run `brew update` before hand to get the latest version of the
+formulae. For example, to install the latest version of Temple:
+
+    brew update
+    brew install temple
+
+To update brew taps:
```
Comment (c-harding) [inline README.md]: can we just simplify this down to

To upgrade to the latest version of Temple:

    brew update
    brew upgrade temple

And skip the general usage instructions for brew?

File: README.md [hunk 15]
```diff
@@ -1,2 +1,65 @@
...
+    brew upgrade temple  
+
+
+## Appendix: overview of installing Homebrew
+
+The Homebrew developers suggest installing Homebrew at `/usr/local` to maximize
+compatibility with existing software. To do so, follow the instructions on
+[their website][brew].
+
+This author prefers `/opt/homebrew`, finding that it works well enough in
```
Comment (c-harding) [inline README.md]: Why do we need our own method of installing brew? The one-liner on the website should be good enough for beginners, and non-beginners will already have it installed

**Review by c-harding — APPROVED** [review]
Well done

**Review by jaylees14 — APPROVED** [review]

**Pull request merged**